Cloud Application Manager/Integrating with Jenkins/jenkins-cloud-application-manager-slaves.md

```diff
@@ -1,15 +1,16 @@
 {{{
 "title": "Launching Slaves via Cloud Application Manager",
-"date": "10-01-2018",
+"date": "01-17-2019",
 "author": "Modified by Gavin Lai",
 "attachments": [],
 "contentIsHTML": false
 }}}
 
 Slave nodes run Jenkins tasks on remote machines in any cloud. If you configure cloud plugins like AWS to launch slaves and then write command line scripts to set up slave build environments then Cloud Application Manager and the Cloud Application Manager Jenkins plugin can save you the hassle of both tasks.
-By configuring the slave node once in Cloud Application Manager, you can reuse it across development, testing, staging, and production. Using Cloud Application Manager, you can launch slaves on any OS, flavor, and cloud. As an example, watch this video on how we use Jenkins slaves to run tests for a sample JBoss app.
+By configuring the slave node once in Cloud Application Manager, you can reuse it across development, testing, staging, and production. Using Cloud Application Manager, you can launch slaves on any OS, flavor, and cloud.
+[comment]: <> (Remove due to EB reference: As an example, watch this video on how we use Jenkins slaves to run tests for a sample JBoss app.)
```

Comment:
`[comment]` is syntax for a link.  That is why the checker is failing.  It thinks this is a link.
Maybe you want **comment** instead?